src/web/pages/reports/auditreportstable.jsx

```diff
@@ -0,0 +1,130 @@
+/* SPDX-FileCopyrightText: 2024 Greenbone AG
+ *
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+
+import React from 'react';
+
+import {_, _l} from 'gmp/locale/lang';
```

Comment:
Using these HOCs we can't avoid the `_l` lazy translation mechanism. It would require some additional refactoring. But we can avoid the standard `_` translation function.
```suggestion
import {_l} from 'gmp/locale/lang';
```

Reply:
Done.